### ISSUE TITLE
http.headers.Clear-Site-Data - Supported in Safari

### ISSUE THREAD

**Issue by JannisBush** [mdn/browser-compat-data#21208]
### What type of issue is this?

Incorrect support data (example: BrowserX says "86" but support was added in "40")

### What information was incorrect, unhelpful, or incomplete?

Table says not supported in Safari, however it is now supported in Safari: https://bugs.webkit.org/show_bug.cgi?id=251856

### What browsers does this problem apply to, if applicable?

Safari

### What did you expect to see?

Support in recent versions (https://bugs.webkit.org/show_bug.cgi?id=251856)

### Did you test this? If so, how?

- Set a cookie on a controlled site
- Send a response with `clear-site-data="*"`
- Cookie was deleted

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

https://bugs.webkit.org/show_bug.cgi?id=251856

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Clear-Site-Data

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `http.headers.Clear-Site-Data`
* Report started: 2023-11-10T08:23:31.150Z

</details>

**Comment by hamishwillee** [comment]
I can see this header is definitely supported according to the preview notes, at least for `executionContext` - https://developer.apple.com/documentation/safari-technology-preview-release-notes/stp-release-164/#HTTP and for `clear` https://developer.apple.com/documentation/safari-technology-preview-release-notes/stp-release-156/#Web-API

@queengooborg How do we map these versions to safari releases? I don't know how to test the other directives.

**Comment by queengooborg** [comment]
> How do we map these versions to safari releases?

We used to keep track of the mapping in #2991, but since `preview` was added (and the collector is able to handle most features), we've stopped tracking the mapping.  It's typically better to go off of WebKit versions nowadays.

I see that the bug links to https://github.com/WebKit/WebKit/pull/9759, which landed in https://github.com/WebKit/WebKit/commit/43c15c5af20a3c4c44e833454113502ecb71a52c.  The commit tags `WebKit-7616.1.2`, which tells me that the feature landed in WebKit version 616.1.2 (the first number is presumably for the OS version, but that's unconfirmed).
